ui/components/src/FormatControls/FormatControls.tsx

```diff
@@ -52,13 +52,13 @@ function getOptionByDecimalPlaces(decimalPlaces?: number) {
   return DECIMAL_PLACES_OPTIONS.find((o) => o.decimalPlaces === decimalPlaces);
 }
 
-export function UnitSelector({ value, onChange, disabled = false }: UnitSelectorProps) {
```

Comment:
I would have kept the word Selector as I feel it's the same meaning than DatasourceSelector

+ Controls will come in force with the validation use case

Reply:
after discussion, nevermind it has nothing to do with DatasourceSelector